src/main.rs

```diff
@@ -68,6 +68,10 @@ fn main() {
         draw_ui(ui_net_info, display_running).expect("Error!");
     });
 
-    network_sniffer_thread.join().unwrap();
+    let res_net = network_sniffer_thread.join();
     display_thread.join().unwrap();
+    if res_net.is_err() {
+        println!("Please run with \"sudo\" if not already running with \"sudo\"");
```

Comment:
Actually we are not sure if the thread failed because the command was not run with `sudo`.
Probably print a more generic error message like -
```suggestion
        println!("Failed to start packet capture.");
```

Reply:
Alright.

Reply:
Made the change in the most recent commit.